core/meterer/util.go

```diff
@@ -0,0 +1,164 @@
+package meterer
+
+import (
+	"context"
+	"math/big"
+
+	commonaws "github.com/Layr-Labs/eigenda/common/aws"
+	test_utils "github.com/Layr-Labs/eigenda/common/aws/dynamodb/utils"
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+)
+
+func DummyCommitment() core.G1Point {
```

Comment:
This doesn't seem used anywhere

Reply:
this is used in later PRs but I've removed it since we are removing commitment from the Payment header